library/stack.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+/** @TODO add this to all methods
+ * Push and pop from a stack
+ * @param {string} value
+ * @param {number} value
+ * @return {array}
+ */
+
+class Stack{
+    //  creating a stack of length = lengthOfStack
+    // creating a new array of the length provided
+    // setting up initial index to 0
+    constructor(lengthOfStack){
+        this.lengthOfStack = lengthOfStack;
+        this.lastUpdatedIndex = 0;
+        this.stack = new Array(this.lengthOfStack)
+    }
+
+    addValueToStack(value){
+    // pushing value onto the stack
+    // if stack is full, throwing an exception
+        if (this.lastUpdatedIndex === this.lengthOfStack){
+            throw new Error ("Error");
+        }
+        this.stack[this.lastUpdatedIndex] = value;
```

Comment:
we are calling the param lastUpdatedIndex but then we are putting the value in the same spot
either we should call it just index or currentIndex or something